pkg/sdn/plugin/pod_linux.go

```diff
@@ -33,7 +33,6 @@ const (
 	sdnScript   = "openshift-sdn-ovs"
 	setUpCmd    = "setup"
 	tearDownCmd = "teardown"
-	statusCmd   = "status"
```

Comment:
Nope, minor cleanup. Since this cleanup is only couple of lines, I have created a separate commit instead of new pr to avoid additional test/merge cycle.